structure_tests/README.md

```diff
@@ -75,7 +77,8 @@ Example:
 		"name": "Root",
```

Comment:
This diff is showing up with tabs instead of spaces, could you keep it as spaces?